@geodata/geodata.m

```diff
@@ -730,13 +730,20 @@
         
         function obj = extractContour(obj,ilev)
             % Extract a geometric contour from the DEM at elevation ilev.
+            % obj = extractContour(obj,ilev)
+            %
+            % Can use to get the mean sea level contour, e.g.;
+            % gdat = geodata('pslg',0,'h0',min_el,'dem',dem); % make the dummy gdat for the dem extents;

```

Comment:
Impressed that this works...I found that it produced disjoint segments that didn't work with the signed distance function too well.

Reply:
It may depend on the complexity of the region. At least for these islands it seems to work really well (and better than using gdal_contour which produced disjoint segments).